src/main/scala/io/movies/model/Movie.scala

```diff
@@ -0,0 +1,5 @@
+package io.movies.model
+
+case class Movie(title: String, director: String, releaseDate: Short)
```

Comment:
It might make more sense to use `LocalDate` for the `releaseDate` field

Reply:
Ok, I have realized what you're storing in the database is just the year. So you could also use `java.time.Year`

Reply:
See https://github.com/gutiory/MoviesServer/pull/1/commits/5f45fab69302e32d3cc3a0a0e695337d0677475c